### PR TITLE
add header-subtitle

### LINKED ISSUE #299: Implement suggestions for better first time user experience
Part of the result of the User Tests #289 should be concrete improvements for first time users.

Coordinate with your other USCC Fellows to discuss possible features and create tickets for them.

### PR DESCRIPTION
**Pull request by mahdikhashan**
resolves #299

### CODE REVIEW

File: boudicca.base/publisher-event-html/src/main/resources/templates/layout/header.hbs [hunk 1]
```diff
@@ -94,6 +110,10 @@
             display: none;
         }
 
+        h3 {
+            display: none;
```
Comment (kadhonn) [inline boudicca.base/publisher-event-html/src/main/resources/templates/layout/header.hbs:114]: what is this css rule about?

File: boudicca.base/publisher-event-html/src/main/resources/templates/layout/header.hbs [hunk 2]
```diff
@@ -139,6 +159,7 @@
                 <img src="/img/logo_small.png" alt="Boudicca Logo - Ein rundes Schwarz-auf-Weiß-Bild. Ein stilisiertes Logo der Mondkrone des keltischen Volksstammes Iceni." height="56px" width="56px"/>
                 <div>
                     <h1 class="header-title">{{title}}</h1>
+                    <h3 class="header-subtitle">All accessible events in Austria</h3>
```
Comment (kadhonn) [inline boudicca.base/publisher-event-html/src/main/resources/templates/layout/header.hbs:162]: 1) While I agree we should add an english version of our website soonish i think we should keep it in german for now so everything is consistent
2) While we have a focus on accessible events, those are not the only ones we have

considering those points maybe we should go with "Events in Österreich und Informationen zu deren Barrierefreiheit"?
it does not sounds as snappy as i would like but currently i have no better suggestion :D

Comment (mahdikhashan) [reply boudicca.base/publisher-event-html/src/main/resources/templates/layout/header.hbs:162]: sure, i agree.

Comment (mahdikhashan) [reply boudicca.base/publisher-event-html/src/main/resources/templates/layout/header.hbs:162]: image

**Review by kadhonn — APPROVED** [review]

**Review by kadhonn — APPROVED** [review]

**Pull request merged**